docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/storage-balance.md

```diff
@@ -41,17 +40,14 @@ Got 3 rows (Time spent: 5886/6835 us)
 
 Create a graph space named **test** with 100 partition and 3 replicas.
 
-```
+```SQL
```

Comment:
all of them need to modify to `nGQL`

Reply:
I'll check how to add a new language specification in a new pr